### ISSUE TITLE
Streaming videos?

### ISSUE THREAD

**Issue by jackfirth** [videolang/video#37]
Could `video` be used to process and render videos in streaming formats such as MPEG-DASH or RSTP? This might work best on top of #36 or some similar mechanism.

(It would be pretty neat to embed the RacketCon youtube livestream in a `#lang slideshow` RacketCon talk this way...)

**Comment by LeifAndersen** [comment]
Yes, this is on the roadmap. I'm not sure if it'll be added to v0.2 or if it will officially slip to v0.3. (Or maybe we could v0.2.1?)

Anyway, thanks for the issue.
